src/resources/armies/moloch.json

```diff
@@ -0,0 +1,332 @@
+{
+  "id": "moloch",
+  "name": "Moloch",
+  "tokens": [
+    {
+      "id": "hq",
+      "hq": true,
+      "initiative": 0,
+      "attacks": [
+        {
+          "type": "riffle",
+          "angle": -1
+        }
+      ],
+      "modifiers": [
+        {
+          "type": "strengthr",
+          "angle": -1,
+          "value": 1,
+          "self": true
+        }
+      ]
+    }, {
+      "id": "blocker",
+      "count": 2,
+      "initiative": 0,
+      "health": 3,
+      "shields": [
+       0 ]
```

Comment:
I guess we could just keep this in the line above.

Reply:
changed as requested